modules/rdma-verifying-ib-connectivity.adoc

```diff
@@ -0,0 +1,174 @@
+// Module included in the following assemblies:
+//
+// * hardware_accelerators/rdma-remote-direct-memory-access.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rdma-verifying-ib-connectivity_{context}"]
+
+= Verifying Infiniband connectivity
```

Comment:
This entire topic needs to be rewritten. @wabouhamad to send new steps to @StephenJamesSmith.